__tests__/app.test.js

```diff
@@ -114,3 +114,61 @@ describe("ALL categories of undefined path", () => {
       });
   });
 });
+
+describe("GET api/reviews/:review_id/comments,", () => {
+  test("Responds with the comments of a specific review ID", () => {
+    const ID = 3;
+    return request(app)
+      .get(`/api/reviews/${ID}/comments`)
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+
+        expect(comments).toEqual([
+          {
+            comment_id: 6,
+            body: "Not sure about dogs, but my cat likes to get involved with board games, the boxes are their particular favourite",
+            review_id: 3,
+            author: "philippaclaire9",
+            votes: 10,
+            created_at: "2021-03-27T19:49:48.110Z",
+          },
+          {
+            comment_id: 3,
+            body: "I didn't know dogs could play games",
+            review_id: 3,
+            author: "philippaclaire9",
+            votes: 10,
+            created_at: "2021-01-18T10:09:48.110Z",
+          },
+          {
+            comment_id: 2,
+            body: "My dog loved this game too!",
+            review_id: 3,
+            author: "mallionaire",
+            votes: 13,
+            created_at: "2021-01-18T10:09:05.410Z",
+          },
+        ]);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
+  });
+
```

Comment:
Would be worth adding another test here, what would happen if the id is valid and exists but there happens to be no comments? This shouldn't be an error in this case, lets test for that circumstance.